cilium/cmd/encrypt_status.go

```diff
@@ -89,13 +90,17 @@ func countUniqueIPsecKeys() int {
 	return len(keys)
 }
 
-func extractMaxSequenceNumber(ipOutput string) string {
-	maxSeqNum := "0"
+func extractMaxSequenceNumber(ipOutput string) int64 {
+	maxSeqNum := int64(0)
 	lines := strings.Split(ipOutput, "\n")
 	for _, line := range lines {
 		matched := regex.FindStringSubmatchIndex(line)
 		if matched != nil {
```

Comment:
nit: perhaps better to reverse the logic here for line of sight ie if no match continue.

Also not keen on bombing on a utility func. Might be better to return the error and let the call site decide on how to proceed from there.

Reply:
Do you mean the fatal by "bombing"? This isn't really new. All our commands currently behave this way.

Reply:
Right! yes I see the pattern and I am personally not fond of it ;( 
I get that this is a limited use case but since we are testing the call might as well test the boundaries without potentially causing the test to bomb too. To boot we could leverage the return error to also tell the caller no max seq number can be found vs making up a value aka "n/a"??